tests/queries/0_stateless/00002_log_and_exception_messages_formatting.reference

```diff
@@ -1,7 +1,7 @@
 runtime messages	0.001
 runtime exceptions	0.05
-messages shorter than 10	1
-messages shorter than 16	3
+messages shorter than 10	2
+messages shorter than 16	4
```

Comment:
It would be better to fix the message itself

Reply:
Done! Added `user_id` to the logout message